x-pack/legacy/plugins/siem/public/containers/case/configure/mock.ts

```diff
@@ -75,7 +76,7 @@ export const caseConfigurationResposeMock: CasesConfigureResponse = {
   created_by: { username: 'elastic', full_name: 'Elastic', email: 'elastic@elastic.co' },
   connector_id: '123',
   connector_name: 'My Connector',
-  closure_type: 'close-by-user',
+  closure_type: 'close-by-pushing',
```

Comment:
`close-by-user` is initialState. Changed to `close-by-pushing ` so we can test changes to `closure_type`